app/controllers/properties_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PropertiesController < ApplicationController
+  def create
+    result = PropertiesCreationService.call(request.raw_post)
+    render json: result.to_h, status: result.success ? 200 : 422
```

Comment:
Like Rob suggested on slack you could try to implement the `to_json` method in the service and then writing just `json: result` here.

Reply:
@Julien - We're going round in circles here.  My first implementation was for the service to generate the json response and https status, and you refactored it so that the controller did this.  I think that is probably better - the service is responsible for creating the records and generating the data that's going into the response, the controller is responsible for formatting it to JSON, XML, HTML or whatever we decide.  I know we're only going to use JSON here, but the principle holds good.

Reply:
are we?